Docs changes for platform (#329)
hirosystems/docs: docs/platform/getting-started.md

---
title: Getting Started
---

## Introduction

> The [Hiro Platform](https://platform.hiro.so/) provides you the opportunity to work with projects and contracts in a cloud-based SaaS environment outside of a normal production environment.

Reviewer: ```suggestion
The [Hiro Platform](https://platform.hiro.so/) allows you to work with Clarity smart contracts in a cloud-based secured development environment.
```

![The Hiro Platform landing page](images/platform-sign-in.png)

## Sign in to the Hiro Platform

When you navigate to the [Hiro Platform](https://platform.hiro.so/), you will be prompted to log into the platform using your login credentials.

If you are a new user, you can sign up to create a new account or continue as an existing user to login with following options.

- Email address

> - Google account

Reviewer: ```suggestion
- Google
```

> - GitHub account

Reviewer: ```suggestion
- GitHub
```

> - Discord account

Reviewer: ```suggestion
- Discord
```

### Sign up as a new user

To sign up as a new user, perform the steps described below.

1. Enter your email address in the "Email address" field.
2. Enter your password in the "Password" field, meeting the password standards as shown in the image below.

![Password standards](images/password-standards.png)

### Sign in using your email address/google/GitHub or discord

To sign into the Hiro platform using your email address, perform the steps described below.

1. Enter your email address in the "Email address" field.
2. Enter your password in the "Password" field.
3. Hiro will attempt to verify your credentials. If successful, you are taken to the [projects](https://platform.hiro.so/projects/) page.

Once you are signed in using any of the above methods, you can follow the create, build, deploy, and monitor sections to work with the Hiro Platform.

## Create or import project

Now that you are signed in using one of the above methods, you can create or import a new project. You can do this by referring to [how to create a project](how-to-guides/how-to-create-project.md).

![create and import project](images/create-or-import-project.png)

## Create and build contracts

> Once you create a project, you can now view the existing contracts in your projects and edit them using the code editor, which is built into the Hiro Platform. You can [create new contracts](how-to-guides/how-to-create-contract.md) by referring to the [contract examples](feature-guides/contract-examples.md) and editing them using the Hiro Platform.

Reviewer: Examples should point to the new MD file that you draft based on https://github.com/hirosystems/clarity-examples/.

> Once you create a project, you can now view the existing contracts in your projects and edit them using the code editor, which is built into the Hiro Platform. You can [create new contracts](how-to-guides/how-to-create-contract.md) by referring to the [contract examples](feature-guides/contract-examples.md) and editing them using the Hiro Platform.

Reviewer: ```suggestion
Once you create or import a project, you can view the existing contracts in your projects and edit them using the VS Code editor available in the Hiro Platform. You can [create new contracts](how-to-guides/how-to-create-contract.md) by referring to the [contract examples](feature-guides/contract-examples.md) and editing them using the Hiro Platform.

```

![create and build contracts](images/vs-code-editor.png)

## Deploy contract

> The Hiro Platform helps you with all the tools required for you to deploy the contracts in less than 5 min. Once your contracts are ready to deploy, follow the [how to deploy contracts] guide to deploy your contracts to testnet or mainnet environments.

Reviewer: ```suggestion
The Hiro Platform helps you with all the tools required to deploy the Clarity smart contracts. Once your Clarity contracts are ready to deploy, follow the [how to deploy contracts] guide to deploy your contracts to testnet or mainnet environments.
```

![Deploy contracts](images/mainnet-deployment.png)

## Monitor contracts

> Once the deployment of the contracts is successful, you can monitor your contracts in Explorer.

Reviewer: ```suggestion
To monitor the progress of your deployed contract, you can use the links to Explorer assigned to each contract and verify the submitted deployment transaction.```

Reviewer: This can include a screenshot to highlight where the links are.
